### ISSUE TITLE
Keep properties-alphabetical-order while giving property-order groups extended rules

### ISSUE THREAD

**Issue by mattfelten** [hudochenkov/stylelint-order#55]
What I'd like to do is be able to group CSS for certain groups, but leave the rest of the declarations in alphabetical order. As an example, I only want `position` lines to be separate.

```
.class {
    color: #000;
    display: block;
    ...alphabetical...
    width: 100%;
    z-index: 100;

    position: absolute;
    top: 0;
    right: 0;
    bottom: 0;
    left: 0;
}
```

I've gotten the group to kind of work with 
```
"order/properties-order": [
	{
		"emptyLineBefore": "always",
		"properties": [
      			"position",
			"top",
			"right",
			"bottom",
			"left"
		]
	}
]
```

But it's saying `Expected position to come before width` so the rule wants the group to still be in order. If I turn off `properties-alphabetical-order` the top area in my example doesn't give me warnings.

Is there a way to do this?

**Comment by hudochenkov** [comment]
Sorry, I don't understand your question.

You can't use `properties-order` and `properties-order-alphabetical` simultaneously.

Have you seen [`unspecified`](https://github.com/hudochenkov/stylelint-order/blob/master/rules/properties-order/README.md#unspecified-topbottombottomalphabeticalignore) optional secondary option for `properties-order`? Maybe it solve your problem.

**Comment by mattfelten** [comment]
After looking at `unspecified`, what I really want is `unspecified: "topAlphabetical"`. I want all unlisted properties to be at the top of the declaration in alphabetical order. Then the specific groups I specified underneath.

**Comment by hudochenkov** [comment]
The only way to achieve this is to add logic for `unspecified: 'topAlphabetical'`. It might be similar to [`unspecified: 'bottomAlphabetical'`](https://github.com/hudochenkov/stylelint-order/blob/4d1a9117b5e440bfc2491bc3f75a26901bdf7223/rules/properties-order/checkOrder.js#L52-L66).

The same option should be added to [postcss-sorting](https://github.com/hudochenkov/postcss-sorting/blob/master/lib/properties-order/unspecified-properties-position.md) as it's used for autofixing. We can't have linting and autofixing have different options.

I have no plans to do this, as it's the first time I see such ordering convention.
